__tests__/app.test.js

```diff
@@ -141,3 +141,57 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  it("should return status 201 and the new comment", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send({
+        username: "butter_bridge",
+        body: "The most amazing comment ever",
+      })
+      .expect(201)
+      .then(({ body: { newComment } }) => {
+        expect(newComment).toMatchObject({
+          author: "butter_bridge",
+          body: "The most amazing comment ever",
+        });
+      });
+  });
+  it("should respond with status 400 bad request if username does not exist", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send({
+        username: "coolDude42",
+        body: "The most amazing comment ever",
+      })
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad request");
+      });
+  });
```

Comment:
Nice error handling test, but this should be a 404 instead of a 400 status code. The reason for this is that the username `"coolDude42"` is a valid username but one that does not exist.